webhook-connector/setting-up-the-webhook.md

```diff
@@ -14,86 +14,87 @@ Self-signed certificates are NOT supported
 {% endhint %}
 
+{% hint style="warning" %}
+When you **use the shipment event** it will contain **ALL** the lines and **ALL** the documents. 
+You can filter new/changed lines and documents based on the `lastChangedAt` fields.
+{% endhint %}
+
+## Setting up the GET order webhook
 
-{% api-method method="get" host="https://yourcompany.com" path="/any/path/:orderId" %}
-{% api-method-summary %}
-Webhook with id
-{% endapi-method-summary %}
-
-{% api-method-description %}
-The GET webhook method is used to send the order id or shipment id when the order or shipment is new or has been updated.
-{% endapi-method-description %}
-
-{% api-method-spec %}
-{% api-method-request %}
-{% api-method-path-parameters %}
-{% api-method-parameter name="orderId" type="string" required=false %}
-order identifier
-{% endapi-method-parameter %}
-{% api-method-parameter name="shipmentId" type="string" required=false %}
-order identifier
-{% endapi-method-parameter %}
-{% endapi-method-path-parameters %}
-{% endapi-method-request %}
-
-{% api-method-response %}
-{% api-method-response-example httpCode=200 %}
-{% api-method-response-example-description %}
-{% endapi-method-response-example-description %}
-{% endapi-method-response-example %}
-{% endapi-method-response %}
-{% endapi-method-spec %}
-{% endapi-method %}
+Your GET order webhook must implement the GET order webhook OpenAPI specification: 
 
 {% hint style="info" %}
-[GET order webhook OpenAPI specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-client/specs.yaml#/order-webhook%20endpoints/webhookGet)
+[GET order webhook OpenAPI specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookGet)
+{% endhint %}
+
+You must also implement the GET order OpenAPI specification to fetch the actual order: 
 
+{% hint style="info" %}
+[GET order OpenAPI specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order/specs.yaml#/order/getOrderByIdRoute)
+{% endhint %}
+
+{% hint style="warning" %}
+When you **GET the order** you will get **ALL** the lines. 
+You can filter new/changed order lines based on the `lastChangedAt` fields.
+{% endhint %}
+
+## Setting up the GET shipment webhook
+
+Your GET shipment webhook must implement the GET order webhook OpenAPI specification: 
+
+{% hint style="info" %}
 [GET shipment webhook OpenAPI specification](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/shipment-webhook-connector/specs.yaml#/shipment-webhook%20endpoints/webhookGet)
 {% endhint %}
 
+{% hint style="warning" %}
+When you **GET the shipment** you will get **ALL** the lines. 
+You can filter new/changed shipment lines and documents based on the `lastChangedAt` fields.
+{% endhint %}
+
+You must also implement the GET shipment OpenAPI specification to fetch the actual order: 
```

Comment:
Guess this should be "the actual shipment"

Reply:
Fixed